src/versioned_db.rs

```diff
@@ -408,49 +471,165 @@ where
     }
 
     /// Returns the value of a key in the live column family.
-    pub fn get_live_value(&self, encoded_schema_key: &[u8]) -> anyhow::Result<Option<V::Value>> {
-        self.live_db.get_raw::<V>(encoded_schema_key)
+    pub fn get_live_value(&self, key: &V::Key) -> anyhow::Result<Option<V::Value>> {
+        let lock = self.cache.try_read();
```

Comment:
`SCHEMADB_GET_LATENCY_SECONDS` has changed in meaning. Previously, it included the cache but now the cache logic has moved here while the metric remains in `get_raw_with_cf_and_decoder`. Is this ok?

Reply:
Good catch. Yes, I think this is fine. We track metrics at the SDK level which include the cache time, and it's nice to have a clean separation between time spent waiting on I/O (measured by `SCHEMADB_GET_LATENCY_SECONDS`) vs time spent waiting on the cache lock.